src/components/group.ts

```diff
@@ -69,6 +69,10 @@ export module Components {
       return this._components.slice();
     }
 
+    /**
+     * Adds a Component to this Group.
+     * The added Component will be rendered on top of Components already in the Group.
```

Comment:
Not sure if the concept of `top` is proper here. I believe `above` (if that is the case) is mor appropriate